src/Cutscenes.ts

```diff
@@ -1637,6 +1637,193 @@ export class Cutscenes<TEightBittr extends FullScreenPokemon> extends EightBittr
     }
 
     /**
+     * Cutscene for a Quick Attack attack in battle.
+     * 
+     * @param settings   Settings used for the cutscene.
+     * @param args   Settings for the routine.
+     */
+    public cutsceneBattleAttackQuickAttack(settings: IBattleCutsceneSettings, args: IBattleAttackRoutineSettings): void {
+        const attackerName: string = args.attackerName;
+        const defenderName: string = args.defenderName;
+        const attacker: IThing = this.EightBitter.BattleMover.getThing(attackerName) as IThing;
+        const defender: IThing = this.EightBitter.BattleMover.getThing(defenderName) as IThing;
+        const direction: number = attackerName === "player" ? 1 : -1;
+        const menu: IMenu = this.EightBitter.MenuGrapher.getMenu("BattleDisplayInitial") as IMenu;
+
+        let xvel: number = -7 * direction;
+
+        // Move the attacking pokemon
+        this.EightBitter.TimeHandler.addEventInterval(
+            (): void => {
```

Comment:
I've been going back and forth on preference for inline lambdas, but think I've solidified.

For ones that return `undefined` or `void` (which end up being `undefined`), keep it one line for ease of reading.

```typescript
// More readable
this.EightBitter.TimeHandler.addEventInterval(
    (): void => this.EightBitter.physics.shiftHoriz(attacker, xvel),
    1,
    38);

// Less readable
this.EightBitter.TimeHandler.addEventInterval(
    (): void => {

    },
    1,
    38);
```